src/components/overall-statistics/AmountPerDayChart.vue

```diff
@@ -23,25 +24,24 @@ export default defineComponent({
   setup(props) {
     const { t } = useI18n();
 
-    const gameDayDates = ref<Date[]>(props.gameDays.map((g) => parseJSON(g.date)));
+    const gameDayDates = ref<Date[]>(props.gameDays.map((g) => utcToZonedTime(parseJSON(g.date), "UTC")));
     const gameDayCounts = ref<number[]>(props.gameDays.map((g) => g.gamesPlayed));
 
-    const chartData: ComputedRef<ChartData> = computed((): ChartData => {
-      return {
-        labels: gameDayDates.value,
-        datasets: [
-          {
-            label: t("components_overall-statistics_tabs_playeractivitytab.playersperday").toString(),
-            data: gameDayCounts.value,
-            fill: true,
-            backgroundColor: "rgba(54, 162, 235, 0.2)",
-            borderColor: "rgba(54, 162, 235, 1)",
-            borderWidth: 1.5,
-            tension: 0.4, // Smooth line.
-          },
-        ],
-      };
-    });
+    const chartData = computed<ChartData<"line">>(() => ({
```

Comment:
Why use an implicit return statement here? If it's a oneliner function, it's nice, but doesn't improve readability if the function spans multiple lines, in my opinion.

Reply:
Removes a level of indentation, less to think about overall IMO. But fair enough, I can revert that if you prefer.

Reply:
It's a minor gripe, and you probably have more professional experience than me, so if you tell me it's stylistically kosher, I trust you.